src/canvas-force-graph.js

```diff
@@ -48,6 +48,7 @@ export default Kapsule({
     linkVisibility: { default: true, triggerUpdate: false },
     linkColor: { default: 'color', triggerUpdate: false },
     linkAutoColorBy: {},
+    linkLineDash: { default: null, triggerUpdate: false },
```

Comment:
The default can be omitted since `null` is the default value.

Reply:
Done